src/redux/incidents/sagas.js

```diff
@@ -120,7 +120,7 @@ export function* getIncidentsImpl() {
     const baseParams = {
       since: DEBUG_SINCE_DATE ? new Date(DEBUG_SINCE_DATE).toISOString() : sinceDate.toISOString(),
       until: DEBUG_UNTIL_DATE ? new Date(DEBUG_UNTIL_DATE).toISOString() : new Date().toISOString(),
-      include: ['first_trigger_log_entries', 'external_references'],
+      include: ['first_trigger_log_entries', 'external_references', 'custom_fields'],
```

Comment:
Custom field values aren't included in the /incidents endpoint. Would need to GET each incident individually to get the `custom_fields` values, or... get the /incidents endpoint adjusted to include the `custom_fields` include.